AsyncDisplayKit/ASImageNode.mm

```diff
@@ -227,6 +227,12 @@ + (UIImage *)displayWithParameters:(_ASImageNodeDrawParameters *)parameters isCa
   // Use contentsScale of 1.0 and do the contentsScale handling in boundsSizeInPixels so ASCroppedImageBackingSizeAndDrawRectInBounds
   // will do its rounding on pixel instead of point boundaries
   UIGraphicsBeginImageContextWithOptions(backingSize, parameters.opaque, 1.0);
+  
+  // if view is opaque, fill the context with background color
+  if (parameters.opaque && parameters.backgroundColor) {
+    [parameters.backgroundColor setFill];
+    [[UIBezierPath bezierPathWithRect:{ .size = backingSize }] fill];
```

Comment:
Let's use a more efficient method to do this.  Instead of creating a path and requiring CG to understand the shape of the path (which it may be able to determine is a rectangle, and short-circuit...but more likely, is not able to) — let's just call CGContextFillRect() :).